cime_config/namelist_definition_drv.xml

```diff
@@ -8,95 +8,102 @@
   <!--  DRIVER attributes                       -->
   <!-- ======================================== -->
 
-  <entry id="verbosity@driver_attributes">
+  <entry id="Verbosity@driver_attributes" modify_via_xml="ESMF_VERBOSITY_LEVEL">
     <type>char</type>
     <category>nuopc</category>
     <group>DRIVER_attributes</group>
+    <valid_values>off,low,high,max</valid_values>
     <values>
-      <value>$INFO_DBUG</value>
+      <value>$ESMF_VERBOSITY_LEVEL</value>
     </values>
   </entry>
 
-  <!-- NUOPC generic input -->
-
-  <entry id="profiling@driver_attributes">
+  <entry id="Profiling@driver_attributes">
```

Comment:
II think you need the modify_via_xml field here.

Reply:
done.